fixtures/organisation.json

```diff
@@ -0,0 +1,47 @@
+{
+  "meta": {
+    "schemaVersion": "2",
+    "timestamp": "2025-03-25T10:00:00Z"
+  },
+  "data": {
+    "main": {
+      "organisationName": "GreenCycle Ltd",
+      "organisationType": "Reprocessor",
+      "operatorType": "Recycling",
+      "contact": {
+        "fullName": "Jane Smith",
+        "email": "jane.smith@greencycle.co.uk",
+        "phone": "+44 117 123 4567"
+      },
+      "address": {
+        "line1": "123 Recycling Park",
+        "line2": "Unit 4",
+        "townCity": "Bristol",
+        "country": "United Kingdom",
+        "postcode": "BS1 5AA"
+      }
+    },
+    "repeaters": {
+      "directors": [
+        {
+          "fullName": "Alice Johnson",
+          "role": "Managing Director",
+          "email": "alice.johnson@greencycle.co.uk"
+        },
+        {
+          "fullName": "Mark Lee",
+          "role": "Operations Director",
+          "email": "mark.lee@greencycle.co.uk"
+        }
+      ]
+    },
+    "files": {
+      "supportingDocuments": [
+        {
+          "fileId": "org-987-654",
+          "link": "https://forms-designer/file-download/org-987-654"
+        }
```

Comment:
No need for files in this test, there are none in the real form as far as I know